flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -1051,13 +1050,13 @@ protected abstract void internalDeregisterApplication(
 		@Nullable String optionalDiagnostics) throws ResourceManagerException;
 
 	/**
-	 * Allocates a resource using the resource profile.
+	 * Allocates a resource using the worker resource specification.
 	 *
-	 * @param resourceProfile The resource description
+	 * @param workerResourceSpec that describes the to be allocated worker.
```

Comment:
```suggestion
	 * @param workerResourceSpec that describes the to be allocated worker
```

param, return or throws tags are usually not terminated by a period.